### PR TITLE
Add deprecations/removals for Chrome 97.

### PR DESCRIPTION
**Pull request by jpmedley**
Fixes #SOME_ISSUE_NUMBER

Changes proposed in this pull request:

-
-
-

### CODE REVIEW

File: site/en/blog/deps-rems-97/index.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+## Remove SDES Key Exchange for WebRTC
+
+The SDES key exchange mechanism for WebRTC has been declared a MUST NOT in the
+relevant IETF standards since 2013. The SDES specification has been declared
+historic by the IETF. Its usage in Chrome has declined significantly over the
+recent year. Consequently [it is
+removed](https://www.chromestatus.com/feature/5695324321480704) as of Chrome
+97.
+
+## Remove WebSQL in Third-Party Contexts
```
Comment (mihajlija) [inline site/en/blog/deps-rems-97/index.md:30]:
```suggestion
## Remove WebSQL in third-party contexts
```

File: site/en/blog/deps-rems-97/index.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  - removals
+  - chrome-97
+---
+
+{% include 'partials/see-all-dep-rem.md' %}
+
+Chrome 97 beta was released on November 18, 2021 and is expected to become the
+stable version in late December, 2021.
+
+## Remove SDES Key Exchange for WebRTC
```
Comment (mihajlija) [inline site/en/blog/deps-rems-97/index.md:21]:
```suggestion
## Remove SDES key exchange for WebRTC
```

File: site/en/blog/deps-rems-97/index.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+The SDES key exchange mechanism for WebRTC has been declared a MUST NOT in the
```
Comment (mihajlija) [inline site/en/blog/deps-rems-97/index.md:23]: @rachelandrew What does this mean? Should we clarify?

Comment (rachelandrew) [reply site/en/blog/deps-rems-97/index.md:23]: It means it's prohibited - https://www.ietf.org/rfc/rfc2119.txt so I think this is probably fine. We're essentially just backing up why it's being removed.

File: site/en/blog/deps-rems-97/index.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
+---
+title: Deprecations and removals in Chrome 97
+description: >
+  A round up of the deprecations and removals in Chrome 97 to help you plan.
+layout: 'layouts/blog-post.njk'
+date: 2021-11-18
```
Comment (mihajlija) [inline site/en/blog/deps-rems-97/index.md:6]:
```suggestion
date: 2021-11-22
```

**Review by mihajlija — APPROVED** [review]

**Pull request merged**